dcpy/builds/plan.py

```diff
@@ -107,6 +108,19 @@ def plan_recipe(recipe_path: Path, version: str | None = None) -> Recipe:
                 )
     elif version is not None:
         recipe.version = version
+    assert recipe.version is not None
+
+    previous_recipe = publishing.try_get_previous_version(
+        recipe.product, recipe.version
+    )
+    if previous_recipe is not None:
+        if recipe.env is None:
+            recipe.env = {}
+        recipe.env["version_prev"] = previous_recipe.label
+
+    env = recipe.env or {}
+    for key in env:
+        os.environ[key] = env[key]
```

Comment:
To be clear, this doesn't update bash/shell env. Just for use within this specific python call